Documentation/Proposals/0000-proposal-template.md

```diff
@@ -0,0 +1,184 @@
+# Feature name
+
+* Proposal: [ST-NNNN](NNNN-filename.md)
+* Authors: [Author 1](https://github.com/author1), [Author 2](https://github.com/author2)
+* Status: **Awaiting implementation** or **Awaiting review**
+* Bug: _if applicable_ [apple/swift-testing#NNNNN](https://github.com/apple/swift-testing/issues/NNNNN)
+* Implementation: [apple/swift-testing#NNNNN](https://github.com/apple/swift-testing/pull/NNNNN)
+* Previous Proposal: _if applicable_ [ST-XXXX](XXXX-filename.md)
+* Previous Revision: _if applicable_ [1](https://github.com/apple/swift-testing/blob/...commit-ID.../Documentation/Proposals/NNNN-filename.md)
+* Review: ([pitch](https://forums.swift.org/...))
+
+When filling out this template, you should delete or replace all of the text
+except for the section headers and the header fields above. For example, you
```

Comment:
These header sections had a few removed, you can compare against the [SE template file](https://github.com/apple/swift-evolution/blob/main/proposal-templates/0000-swift-template.md) directly